api/src/pcapi/core/users/external/zendesk_sell.py

```diff
@@ -477,7 +477,7 @@ def _stub_in_test_env(action: str, data: offerers_models.Offerer | offerers_mode
         testing.zendesk_sell_requests.append({"action": action, "type": type(data).__name__, "id": data.id})
         return True
 
-    if settings.IS_DEV or not ZENDESK_SELL_API_KEY:
+    if settings.IS_DEV or not settings.ZENDESK_SELL_API_KEY:
```

Comment:
(question) ce bout de code était déjà là avant cette PR, mais je me demande si vérifier `settings.ZENDESK_SELL_API_KEY` est une bonne idée. Si jamais il y a un défaut de configuration sur la prod, on va logger silencieusement au lieu d'échouer bruyamment. Est-ce que ça n'est pas un problème ? J'aurais plutôt en tête quelque chose comme : 

```suggestion
    if settings.IS_DEV or settings.IS_TESTING or settings.IS_STAGING:
```

(à adapter selon les environnements où l'on s'attend à ne pas contacter Zendesk)

Au passage, ça rendra plus explicite qu'on ne fait rien sur testing et staging.

Reply:
On peut faire comme ça, de toute façon on n'a pas d'environnement Zendesk Sell hors prod, donc ça se simplifierait même en :
`if not settings.IS_PRODUCTION:`